e2etest/pkg/service/tweak.go

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package service
+
+import corev1 "k8s.io/api/core/v1"
+
+type Tweak func(svc *corev1.Service)
+
+func TrafficPolicyLocal(svc *corev1.Service) {
+	svc.Spec.ExternalTrafficPolicy = corev1.ServiceExternalTrafficPolicyTypeLocal
+}
+
+func ForceV6(svc *corev1.Service) {
+	f := corev1.IPFamilyPolicySingleStack
+	svc.Spec.IPFamilyPolicy = &f
+	svc.Spec.IPFamilies = []corev1.IPFamily{corev1.IPv6Protocol}
+}
+
+func DualStack(svc *corev1.Service) {
+	f := corev1.IPFamilyPolicyPreferDualStack
+	svc.Spec.IPFamilyPolicy = &f
```

Comment:
we need to set IPFamilies too ?

Reply:
IPFamilies, wasn't sure. PreferDualStack should work but we won't know if it silently fails, so I think require is a better choice. Will change.